lib/src/source/git.dart

```diff
@@ -562,6 +563,28 @@ class BoundGitSource extends CachedSource {
         .run(['checkout', ref], workingDir: repoPath).then((result) => null);
   }
 
+  /// Checks out any LFS binaries at the current ref to [repoPath].
+  /// Ignore errors if `git lfs` is not installed.
+  Future _checkOutLFS(String repoPath, String origin) async {
+    // default to whatever the origin is in the .lfsconfig file
+    var args = ['lfs', 'pull'];
+    try {
+      // if the LFS origin is not set via an .lfsconfig file then use the git origin from pubspec
+      var env = await git.run(['lfs', 'env'], workingDir: repoPath);
+      if (env.indexWhere((e) => e.contains('Endpoint=file')) > -1) {
+        args = ['lfs', 'pull', origin];
+        log.fine(
+            'No Git LFS origin found in .lfsconfig, using origin from pubspec');
+      }
+
+      return git
+          .run(args, workingDir: repoPath)
+          .then((result) => null, onError: (error) => null);
+    } catch (e) {
```

Comment:
Assuming any error to be because `git lfs` is not installed seems a bit problematic. It would be preferable to at-least handle the appropriate exception / exit-code, or check if `git lfs` is installed.